presto-native-execution/presto_cpp/main/tests/QueryContextManagerTest.cpp

```diff
@@ -47,31 +47,39 @@ TEST_F(QueryContextManagerTest, nativeSessionProperties) {
   protocol::TaskId taskId = "scan.0.0.1.0";
   protocol::SessionRepresentation session{
       .systemProperties = {
-          {"native_max_spill_level", "1"},
+          {"native_max_spill_level", "2"},
```

Comment:
Curious: Why did we change this value?

Reply:
Default will be 1, so we need something different here than default.